ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__4.15-nightly.yaml

```diff
@@ -109,7 +105,7 @@ tests:
       ARO_WORKER_VM_SIZE: Standard_D8s_v3
       CHURN: "false"
       GC: "false"
-      ITERATION_MULTIPLIER_ENV: "4"
+      ITERATION_MULTIPLIER_ENV: "19"
```

Comment:
@paigerube14 Any reason why it's set to 19 vs 10 for other configs? Is it because of the instance types we are using for Azure?